### ISSUE TITLE
Issue ID: DAC_Unlabelled_Form_Element1

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#5070]
Report Link: https://drive.google.com/file/d/1qo4MWsCD0KQ1e8n3ZTuGFaOCYPaLDcR5/view?usp=sharing

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by bryophyta** [comment]
>**Note**
>This is an automatically-generated comment. Feel free to edit the content, but please don't edit this 'note' block.
 
# Intro to report

Copying the intro here to aid discoverability of the report's content:
```
Unlabelled Form Element
A button does not have a corresponding label.
WCAG Reference:
1.3.1 Info and Relationships (Level A)
Understanding Info and Relationships | How to Meet Info and Relationships
4.1.2 Name, Role, Value (Level A)
Understanding Name, Role, Value | How to Meet Name, Role, Value
Issue ID: DAC_Unlabelled_Form_Element1
URL: https://www.theguardian.com/politics/audio/2018/mar/14/are-the-eus-draft-tradeguidelines-reasonable-brexit-means-podcast
Page title: Are the EU's draft trade guidelines reasonable? Brexit Means… podcast | Politics
| The Guardian
Journey: 1.5
Screenshot:

Users are presented with a set of buttons to control a podcast; however, these buttons have
not been provided with an appropriate label. Users rely on descriptive labels to help them
understand the purpose of buttons. A missing label can and will affect multiple user groups,
such as, screen reader users who are not provided with any instruction on what the button
is in relation to while navigating the page both in and out of context and voice activation
users, who are not given an accessible label to use to dictate an interaction with the button.
```
